models/engine/file_storage.py

```diff
@@ -68,3 +68,23 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """get an object"""
+        for key in self.__objects:
+            tp, obj_id = key.split('.')
+            if tp == cls.__name__ and obj_id == id:
+                return self.__objects[key]
+        return None
+
+    def count(self, cls=None):
+        """count of cls"""
+        if cls is None:
+            return len(self.__objects)
+
+        count = 0
+        for key in self.__objects:
+            tp = key.split('.')[0]
+            if tp == cls.__name__:
+                count += 1
+        return count
```

Comment:
Overall, the methods do what they're supposed to, but they could be simplified and made more efficient. Instead of checking each object's class name individually, it might be faster to keep track of how many objects there are for each class separately. Or, if you're only interested in one class, you could directly look at the objects for that class instead of going through all of them.